terraform/shared/shared.tf.json.template.py

```diff
@@ -137,12 +198,106 @@
+                        {
+                            'Action': [
+                                's3:*'
+                            ],
+                            'Effect': 'Allow',
+                            'Resource': [
+                                '${aws_s3_bucket.aws_config.arn}',
+                                '${aws_s3_bucket.aws_config.arn}/*'
+                            ]
+                        }
+                    ]
+                })
             }
         },
         'aws_api_gateway_account': {
             'shared': {
                 'cloudwatch_role_arn': '${aws_iam_role.api_gateway.arn}'
             }
+        },
+        'aws_config_configuration_recorder': {
+            'default': {
+                'name': config.qualified_resource_name('awsconfig'),
+                'role_arn': '${aws_iam_role.aws_config.arn}',
+                'recording_group': {
+                    'all_supported': True,
+                    'include_global_resource_types': True
+                }
+            }
+        },
+        'aws_config_configuration_recorder_status': {
+            'default': {
+                'name': '${aws_config_configuration_recorder.default.name}',
+                'is_enabled': True,
+                'depends_on': [
+                    'aws_config_delivery_channel.default'
+                ]
+            }
+        },
+        'aws_iam_role_policy_attachment': {
+            'config': {
+                'role': '${aws_iam_role.aws_config.name}',
+                'policy_arn': 'arn:aws:iam::aws:policy/service-role/AWS_ConfigRole'
+            }
+        },
+        'aws_config_delivery_channel': {
+            'default': {
+                'name': config.qualified_resource_name('awsconfig'),
+                's3_bucket_name': '${aws_s3_bucket.aws_config.bucket}',
+                'depends_on': [
+                    'aws_config_configuration_recorder.default'
+                ]
+            }
+        },
+        'aws_guardduty_detector': {
+            'default': {
+                'enable': True,
+                # All data sources are enabled in a new detector by default.
+                'datasources': {
+                    'kubernetes': {
+                        'audit_logs': {
+                            'enable': False
+                        }
+                    }
+                }
+            }
+        },
+        'aws_securityhub_account': {
+            'default': {}
+
+        },
+        'aws_securityhub_finding_aggregator': {
+            'default': {
+                'linking_mode': 'ALL_REGIONS',
+                'depends_on': [
+                    'aws_securityhub_account.default'
+                ]
+            }
+        },
+        'aws_securityhub_standards_subscription': {
+            'aws_fsbp': {
+                'standards_arn': 'arn:aws:securityhub:us-east-1::standards/'
+                                 'aws-foundational-security-best-practices/v/1.0.0',
```

Comment:
```suggestion
                'standards_arn': 'arn:aws:securityhub:us-east-1::standards'
                                 '/aws-foundational-security-best-practices/v/1.0.0',
```

Here and wherever you wrap string literals at slashes.